handlers.go

```diff
@@ -315,24 +315,19 @@ var defaultFcInitHandlerList = HandlerList{}.Append(
 	ConfigMmdsHandler,
 )
 
-var loadSnapshotHandlerList = HandlerList{}.Append(
-	SetupNetworkHandler,
-	StartVMMHandler,
-	CreateLogFilesHandler,
-	BootstrapLoggingHandler,
-	LoadSnapshotHandler,
-	AddVsocksHandler,
-)
+var loadSnapshotRemoveHandlerList = []Handler{
```

Comment:
Can you please add a comment here for future reference on why we need to have a filter list?